tests/light/src/axosyslog_light/syslog_ng_config/statements/sources/network_source.py

```diff
@@ -65,7 +65,17 @@ def write_log(self, message, rate=None, transport=None, framed=None):
         self.io.write_messages([message], rate=rate, transport=transport, framed=framed)
 
     def write_logs(self, messages, rate=None, transport=None, framed=None):
-        self.io.write_messages(messages, rate=rate, transport=transport, framed=framed)
+        loggen_process = self.io.write_messages(messages, rate=rate, transport=transport, framed=framed)
+        loggen_process.wait()
```

Comment:
Now that we made the `write_logs()` call a blocking function, shouldn't we replace all the uses of it to the async counterpart? With the current state of the PR we changed the behavior of all the testcases.

Reply:
The original idea was that `write_logs()` will return when all the logs are written due to actually we write only a few logs in each testcase, what does not need extra time. This change has no effect for the current Light testcases.
In my opinion `write_logs_async()` like functions can be used only in special cases when we want to highlight that the write task should work in background. This is how we can differentiate use-cases.

Reply:
I don't fully understand this comment. @mitzkia please press resolve if you believe this has been addressed :)